### PR TITLE
#784 fix with inline array of type array<K, V>

### PR DESCRIPTION
**Pull request by aviortm**

### CODE REVIEW

File: tests/JMS/Serializer/Tests/Fixtures/ObjectWithInlineArray.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public $array;
+
+    /**
+     * @param array $array
+     */
+    public function __construct(array $array)
+    {
+        $this->array = $array;
+    }
+
```
Comment (goetas) [inline tests/JMS/Serializer/Tests/Fixtures/ObjectWithInlineArray.php]: too many empty lines

Comment (aviortm) [inline tests/JMS/Serializer/Tests/Fixtures/ObjectWithInlineArray.php]: fixed

**Comment by AlexeyDsov** [comment]
Could you merge this fix to 1.7 version?

**Comment by goetas** [comment]
since is not a bugfix, but more an enhancement, will land in 1.8.0

**Comment by aviortm** [comment]
@goetas 
Hi,
I removed the merge commit. Is it ok now?

**Comment by AlexeyDsov** [comment]
> since is not a bugfix, but more an enhancement, will land in 1.8.0

@goetas this case became broken in 1.7 version after commit [32f0e131](https://github.com/schmittjoh/serializer/commit/32f0e1318c183c302e64bac6f12140f78a8feb16)
i.e. this works before 1.7 and after 1.7 but never will work in 1.7?

**Comment by goetas** [comment]
@AlexeyDsov ah... I see.

**Comment by goetas** [comment]
thanks!

**Pull request merged**